Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -185,7 +185,7 @@ private void InitializeNotifyIcon()
             var setting = items.Add(InternationalizationManager.Instance.GetTranslation("iconTraySettings"));
             setting.Click += (o, e) => App.API.OpenSettingDialog();
             var exit = items.Add(InternationalizationManager.Instance.GetTranslation("iconTrayExit"));
-            exit.Click += (o, e) => Close();
+            exit.Click += (o, e) => Environment.Exit(0);
```

Comment:
For this one, don't change. Calling Close() will allow saving normal process of saving setting.

Reply:
OK. fixed.

Reply:
When executing 'open maintaining folder' on the file plug-in, the problem in which 'lastemptyquery' does not work is also fixed to this pr.

Reply:
@onesounds should this be Close() not environment.exit acording to @taooceros ?

Reply:
yes. If I use "environment.ext" here, the process ends immediately, so it don't have time to save settings. If I do as Taoo said, close() will call onclosing(), so it will exit after the save is completed.

This part is not related to this pr topic, just there is a problem that the process continues to remain, so I applied it.